conda-forge.yml

```diff
@@ -7,3 +7,9 @@ travis:
   secure: {BINSTAR_TOKEN: nNvWQJrxMuAYKMLCzuq+rrrmWF0V/den+n1wJHQ7zU7ix+jbNrpp7m9SosLhBL+EhSJ62cguNtrrA9+MMO/ZJx52Lg/Cm9MTBUoCkZgyaUdrXhoqcCupXmFM2SEirIng7Dw4q+qsQSx8/7ugbphCMmp1YtM6GuY6lQYSBnTJ0NN9OjP1WlpbtFar6HGUuk9uOcjlo4gEFPAvGU+wLIZN9q3HvKMuQjvd624fCXeZnCSMULzcOaIcCq1f2aQ04zS9hL6hgxtiARMnFhykvR6bDtCJOrAP0+mg6n2Yo9jmk7UnNCLtxe+lbxtw/q5hybzjGnoLe6h4yNsdyPEfxqIVM1Wg4oz78wNdDIAKbJAtGcUc3LbyOcNFdM9DuXsIM201d4dZjAvkZtRtWxWvdQcwX1oaAPfrFFZcqBoq/pHEAQUteX7UTEjipA1zWVU4VxHFqhmXoR8zwb/2jTF75jmFDKqvf6FdbW3/1tLdIkDSVJTKxznfwAcOznosjlwrCavKYYCkM6+d4h2dX9ZCg9AtfeUWgYC46OsGYcLZAPZVEEL7p6mSmEC5Hjcjv1kuAqoluVzmEF8cdgHKwp8E9FDVOL/lUlji+hZThzg3OtQfx5UjMUIEbZ4a0SdtMhRrDJSeOyUblGx0A1J210+PSHjciI+WxlgxJ5QuKb9qoYoS78w=}
 provider:
   win: azure
+channels:
+  sources:
+    - conda-forge
+    - defaults
+  targets:
+    - ["conda-forge", "rc"]
```

Comment:
@ocefpaf Is this the correct way to ensure that the built release-candidate is pushed to the `conda-forge/labels/rc` channel on anaconda?

Reply:
Nope. Let me update that...

Reply:
Also, we'll need a dev branch for it.

Reply:
Ahh nice one, thanks for clarifying :+1:

Reply:
@ocefpaf I noticed in cfep-05 it gives the example as `rc_matplotlib` but on the conda-forge anaconda cloud repo there is the label `matplotlib_rc`. Does that matter? Or is the protocol to just stick to the order stated in the cfep?

Also, why do some package release into the `rc` channel like `pandas`, `matplotlib` and `zarr`?

Apologies for so many questions :blush:

Reply:
All of those are from before the CEP-05. Actually, `iris` is probably the first one to be CFEP-05 compliant. We have a long way ahead to fix the exiting dev releases.

Reply:
Ahhh cool, I understand now thanks :smile: 

Thanks for getting back to me @ocefpaf, much appreciated indeed :tada: